pkg/log/options.go

```diff
@@ -84,12 +84,12 @@ type Options struct {
 	ErrorOutputPaths []string
 
 	// RotateOutputPath is the path to a rotating log file. This file should
-	// be automatically rotated over time, based on the rotation parameters such
-	// as RotationMaxSize and RotationMaxAge. The default is to not rotate.
+	// be automatically rotated over time, based on the rotation parameter:
+	// RotationMaxSize.
```

Comment:
why delete `RotationMaxAge`?

Reply:
RotationMaxAge does not affect file rotation, just affect file backups instead.

Reply:
Any more questions? cc @hzxuzhonghu 

The comments is wrong, according to [lumberjack](https://github.com/natefinch/lumberjack/blob/v2.0/lumberjack.go#L89-L94):
```
// MaxAge is the maximum number of days to retain old log files based on the
// timestamp encoded in their filename
```

Reply:
Donot we call storeing logs with timestamp as rotation? I think this is also a kind of rotation

Reply:
okay, seems make sense too, just reverted it.